settings.gradle

```
@@ -19,7 +19,9 @@ include 'jaxrs-bom'
 include 'jaxrs-processor'
 include 'jaxrs-server'
 include 'jaxrs-server-security'
-include 'jaxrs-servlet'
+//include 'jaxrs-servlet'
```

did you intentionally remove the `jarxs-servlet`module ?

```suggestion
include 'jaxrs-servlet'
```

---

Needs a new release of servel with fixed message body handlers